docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+# WARNING!
+# This is a development version of the docker-compose.yml file.
+# Avoid using this file in your production environment.
+# We're exposing here sensitive ports and mounting code volumes for rapid development and debugging of the server stack.
+
+version: '3'
+
+services:
+  openruntimes-proxy:
+    container_name: openruntimes-proxy
+    build:
+      context: .
+    networks:
+      servers:
+    ports:
+      - 9800:80
+    volumes:
+      - ./app:/usr/local/app:rw
+      - ./tests:/usr/local/tests:rw
+    environment:
+      - OPEN_RUNTIMES_PROXY_ALGORITHM
+      - OPEN_RUNTIMES_PROXY_EXECUTORS
+      - OPEN_RUNTIMES_PROXY_PING_INTERVAL
+      - OPEN_RUNTIMES_PROXY_VERSION
+      - OPEN_RUNTIMES_PROXY_ENV
+      - OPEN_RUNTIMES_PROXY_EXECUTOR_SECRET
+      - OPEN_RUNTIMES_PROXY_SECRET
+      - OPEN_RUNTIMES_PROXY_LOGGING_PROVIDER
+      - OPEN_RUNTIMES_PROXY_LOGGING_CONFIG
+  mockoon1:
```

Comment:
Once executor is part of OPR, we should just use it. Btw, how is Mockoon? Can we use it for s3 and other kind of tests?

Reply:
Mockoon is a simple HTTP server, pretty much. You can configure routes to give specific responses based on simple JSON config file. It also brings some dynamic parts, for example, it generated a random number from 0 to 100 for CPU usage for health endpoint.

It's something I use for now until the executor is finished.

Not sure what you mean by s3 tests.

Reply:
S3 for our storage library

Reply:
Not sure how they support file upload. It's mainly a JSON server. Tho, I can imagine we could use MinIO for S3 tests.